apps/shim-clr-ui/angular.json

```diff
@@ -31,8 +34,6 @@
             "inlineStyleLanguage": "scss",
             "assets": ["src/favicon.ico", "src/assets"],
             "styles": [
-              "node_modules/@cds/core/global.min.css",
```

Comment:
I'm pretty sure we've discussed this and I apologize if I'm retreading those discussions. But why are we removing `global.min.css` again?

Reply:
As this is just `@clr/angular` app, currently it doesn't have anything that depends on global.min.css.